services/logging-operator/5.2.0/extra-images.txt

```diff
@@ -1,4 +1,4 @@
 {{ .Values.fluentbit.image.repository }}:{{ .Values.fluentbit.image.tag }}
 {{ .Values.fluentd.image.repository }}:{{ .Values.fluentd.image.tag }}
-ghcr.io/kube-logging/config-reloader:v0.0.6
-ghcr.io/kube-logging/node-exporter:v0.8.0
+ghcr.io/kube-logging/config-reloader:v0.0.7
```

Comment:
https://kube-logging.dev/docs/image-versions/#logging-operator-version-52 this comes with 
```
ghcr.io/kube-logging/logging-operator/config-reloader:5.2.0
```